File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    while (true)
    {
        n = get_int("Height (between 1 and 8): ");

        if (n >= 1 && n <= 8)
        {
            break;
        }
    }

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
        if 1 <= height <= 8:
            break
        else:
            print("Enter a number between 1 and 8")
    except ValueError:
        print("Enter a positive number")

for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")

    for k in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for k in range(i + 1):
        print("#", end="")

    print()
